bucket/starship.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.26.3",
+    "description": "The minimal, blazing fast, and extremely customizable prompt for any shell!",
+    "homepage": "http://starship.rs",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/starship/starship/releases/download/v0.26.3/starship-x86_64-pc-windows-msvc.zip",
+            "hash": "6d7729acba118432501032876dbdb9ab0867921e5fa599e77aed1e709569981d"
+        }
+    },
+    "bin": "starship.exe",
```

Comment:
```suggestion
    "extract_dir": "target\\x86_64-pc-windows-msvc\\release",
    "bin": "starship.exe",
```

![A](https://i.imgur.com/6u6FIAk.png)

Reply:
Good catch! It looks like an oversight in our automated release process.
I'll be fixing it and making a new release to address this.

Reply:
The binary should no longer be nested within folders in the archive. 👍